apps/client/src/pages/race/templates/race-text/RaceText.template.tsx

```diff
@@ -280,6 +300,10 @@ export function RaceText({
                     charIndex,
                     { cursorAt: playerCursorAt },
                   );
+                  const isCursorAtInvalidCursor = indexConverter.isCursorAtChar(
```

Comment:
this has Cursor twise in it's name  should be `isCursorAtInvalidCharacter` right ?

Reply:
renamed in #114 a2f934c4d69cdfed5c031f3d9174c8e4a76330e6